### DISCUSSION TITLE
integration with hatch

### DISCUSSION THREAD

**Discussion by gokulyc** [pypa/pipenv#5418] [General]
Is there any plan to merge hatch & pipenv into one?

**Comment by kalebmckale** [comment]
I'm not sure the intersection between the developers of `hatch` and `pipenv` contains many, if any, individuals. Did you pose the question to its developers and get a response?

**Reply by gokulyc** [reply]
How to build wheels using pipenv?

**Reply by kalebmckale** [reply]
I'd say that's outside the scope of the purpose of this project.

**Reply by kalebmckale** [reply]
I haven't used `hatch` before, but with `build` and `twine` as dev packages, I added build and deploy scripts to my Pipfile.
